packages/@react-spectrum/tooltip/test/TooltipTrigger.test.js

```diff
@@ -34,10 +34,6 @@ describe('TooltipTrigger', function () {
         </Provider>
       );
 
-      expect(() => {
```

Comment:
why was this removed?

Reply:
is this because of this comment?
> Still not sure why the closing actions aren't being registered in tooltiptrigger tests file 🤔 so I deleted them instead of doing the id checking hack. Technically this is handled with the tooltip manager tests file anyway 🤷‍♂

Lets try to figure out why, I have some time today, maybe we can look at it together after lunch?

Reply:
No I just didn't think that was a necessary part of that test. I'll add it back

Reply:
O actually that did cause a failure... not sure why